docs/Developers/Newcomers_Guide.rst

```diff
@@ -1,6 +1,10 @@
 Welcome to the Newcomers' Guide!
 ================================
 
+.. container:: shorturl
+
+    Short URL: https://coala.io/newcomer, https://coala.io/newcomers
```

Comment:
IMO this is not the proper ```shorturl``` that is being asked for in the issue. Though I'm not much sure about it so let us wait for other reviewers comments.

Also, design wise it is not looking good. You may want to modify it a bit

Reply:
@shikharvaish28 Can you provide me with a sample design?

Initially I was also not sure, but please check here: https://github.com/coala/documentation/issues/252#issuecomment-284202727